LibertyDSNP/parquetjs
# Feature: Frequency Schema Builder

Problem
=======
Adds a Frequency Schema Builder to support easy building of parquet files from https://github.com/LibertyDSNP/schemas/

Closes: #73 

In general expands the data that can easily build a Parquet file.

Change summary:
---------------
* Renamed result -> snapshot for the simple snapshot test
* Added Frequency Schema Support

Steps to Verify:
----------------
1. Grab a Parquet schema from https://github.com/LibertyDSNP/schemas/
2. `const [parquetSchema, writerOptions] = ParquetSchema.fromFrequencySchema(frequencySchema);`
3. Now you have the schema as well as the writer options to use. See the test for a full example.
